spec/unit/plugins/darwin/virtualization_spec.rb

```diff
@@ -0,0 +1,108 @@
+#
+# Author:: Pavel Yudin (<pyudin@parallels.com>)
+# Copyright:: Copyright (c) 2015 Pavel Yudin
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require File.expand_path(File.dirname(__FILE__) + '/../../../spec_helper.rb')
+
+describe Ohai::System, "Darwin virtualization platform" do
+  before(:each) do
+    @plugin = get_plugin("darwin/virtualization")
```

Comment:
We prefer to use `:let` instead of instance variables in `before(:each)` blocks. Check out the "Use let and let!" subsection of [Better Specs](http://betterspecs.org/).